django/db/backends/base/features.py

```diff
@@ -342,6 +342,8 @@ class BaseDatabaseFeatures:
     json_key_contains_list_matching_requires_list = False
     # Does the backend support JSONObject() database function?
     has_json_object_function = True
+    # Does the backend support using double quotes in JSON field lookups?
+    supports_double_quotes_in_json_field_lookups = True
```

Comment:
I don't think we need a separate feature flag, I'd skip new tests in `DatabaseFeatures.django_test_skips` on SQLite backend.

Reply:
Thanks, I agree that a feature flag is not needed 👍 

I think we should use `django_test_expected_failures` instead of `django_test_skips`, so that we will be "notified" if SQLite fixes the issue in a newer version (that the CI uses). I've updated the commit to do so.